lib/verify.py

```diff
@@ -22,7 +22,106 @@ def verify_same_docs(expected_num_docs, doc_dict_one, doc_dict_two):
     print(" -> doc_dict_one == doc_dict_two expected (num_docs: {})".format(expected_num_docs))
 
 
-def verify_changes(users, expected_num_docs, expected_num_revisions, expected_docs):
+def verify_docs_removed(users, expected_num_docs, expected_docs):
```

Comment:
this function should verify that tombstone revision doesn't change after the doc has been marked deleted.